pest-management-science.csl

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>Pest Management Science</title>
+    <title-short>PMS</title-short>
+    <id>http://www.zotero.org/styles/pest-management-science</id>
+    <link href="http://www.zotero.org/styles/pest-management-science" rel="self"/>
+    <link href="http://www.zotero.org/styles/spie-proceedings" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1526-4998/homepage/ForAuthors.html#preparing" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="biology"/>
+    <issn>1526-498X</issn>
+    <eissn>1526-4998</eissn>
+    <updated>2017-08-13T11:07:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="available at">available</term>
+    </terms>
+  </locale>
+  <macro name="type-number">
+    <group delimiter=" ">
+      <choose>
+        <if type="patent" match="none">
```

Comment:
what's the purpose of this? If anything, wouldn't we expect "US Patent" to be in the genre field for patents (e.g. in Mendeley; Zotero doesn't have one)?